src/main/java/com/javabom/baseballgame/domain/domain/result/MatchedResult.java

```diff
@@ -0,0 +1,19 @@
+package com.javabom.baseballgame.domain.domain.result;
+
+public enum MatchedResult {
+    STRIKE, BALL, OUT;
+
+    public static MatchedResult findResult(final boolean allMatchStatus, final boolean anyMatchStatus) {
+        if (allMatchStatus) {
+            return STRIKE;
+        }
+        return findAny(anyMatchStatus);
+    }
+
+    private static MatchedResult findAny(final boolean anyMatchStatus) {
+        if (anyMatchStatus) {
+            return BALL;
+        }
+        return OUT;
+    }
```

Comment:
```suggestion
    public static MatchedResult findResult(final boolean allMatchStatus, final boolean anyMatchStatus) {
        if (allMatchStatus) {
            return STRIKE;
        }
        if (anyMatchStatus) {
            return BALL;
        }
        return OUT;
    }
```
이건 굳이 메소드 나눌필요가 없을듯??

Reply:
인텐트 문제때문에 나눴는데 아닌가??

Reply:
인덴트에 문제없어보이는데 ??

Reply:
아 이거 인텐트 1이구나!